src/components/DOW/PricingDetails.vue

```diff
@@ -3,9 +3,9 @@
     <ATATRadioGroup 
       id="CurrentPaymentArrangement"
       legend="Current payment arrangement"
-      tooltipText="<strong>Reserved instances</strong> allow you to save on usage 
-        costs for a fixed term. <strong>Pay-as-you-go instances</strong> let you 
-        pay for compute capacity by the hour or second with no long-term commitments."
+      tooltipText="<b>Reserved/pre-paid/up-front instances</b> allow you to save on 
```

Comment:
please change to `<strong>` 
https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb

Reply:
done